pkg/minikube/cluster/cluster.go

```diff
@@ -132,18 +132,18 @@ func DeleteHost(api libmachine.API) error {
 	return m.ToError()
 }
 
-// GetHostStatus gets the status of the host VM.
-func GetHostStatus(api libmachine.API) (string, error) {
+// GetHostStatus gets the status of the host VM with the specified name.
+func GetHostStatus(api libmachine.API, machine string) (string, error) {
```

Comment:
Initially we had planned that we will not change this as it would make difficult for us to use Minikube as a pkg. Hence we had https://github.com/minishift/minishift/issues/1402. I wanted to give a serious try to use minikube as a lib. So little skeptical with this change.

Reply:
Sure, I am aware of this. However, this is a minor change and when it comes to #1402 we can always write a couple of adapters. In fact we will need to do this anyways. The improved code structure outweighs imo this particular change. 

I can later also explain what I mean with adapters. I think this be the best way to address #1402.

Reply:
An abstraction (or override) can be used to feed the function with the constant if needed. So this change is minimal.